internal/store/postgres/appeal_repository.go

```diff
@@ -87,7 +87,7 @@ func (r *AppealRepository) Find(ctx context.Context, filters *domain.ListAppeals
 
 func (r *AppealRepository) GetAppealsTotalCount(ctx context.Context, filter *domain.ListAppealsFilter) (int64, error) {
 	db := r.db.WithContext(ctx)
-	db = applyAppealFilter(db, filter)
+	db = applyAppealFilter(db, filter, false)
```

Comment:
instead of adding a flag here, we can create a copy of `filter` and remove the `size` and `offset` from the struct so it'll be straightforward that in `GetAppealsTotalCount` we are removing those filters